src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -132,6 +137,18 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .WithDescription("A space seperated list of functions to load.")
                 .Callback(f => EnabledFunctions = f);
 
+            Parser
+                .Setup<bool>('q', "quite")
```

Comment:
@anthonychu - can you ACK on  command line option - `quite`?

@fabiocav / @anthonychu  - Do we want `quite` to be the default experience? As a first step, can we make this explicit and then make it default in the next release?